pointmatcher/DataPointsFilters/Geometry.h

```diff
@@ -0,0 +1,80 @@
+// kate: replace-tabs off; indent-width 4; indent-mode normal
+// vim: ts=4:sw=4:noexpandtab
+/*
+
+Copyright (c) 2010--2018,
+François Pomerleau and Stephane Magnenat, ASL, ETHZ, Switzerland
+You can contact the authors at <f dot pomerleau at gmail dot com> and
+<stephane at magnenat dot net>
+
```

Comment:
I would suggest asking the author if you can also add yourself here, not only to ack your contribution, but as a way of informing people of who wrote the file, and maybe providing contact information and a mention of the papers where this method is described.

Reply:
I was actually about to put myself there (in one commit, I had there TODO), but then all the files, even written by our students, have this copyright header. So I was not sure if by changing it, I would not have messed some legal stuff (there is the 2010--2018, so if I put myself there, do I have to somehow extend the copyright or not...). @pomerlef How to add note about the author? Below the copyright text?